lib/src/components/categoryCreateModal/categoryCreateModal.dart

```diff
@@ -28,6 +30,10 @@ class CategoryCreateModalComponent {
 
   List<String> colors = CAT_COLORS;
 
+  void ngOnInit() {
```

Comment:
Does this work on subsequent creates? So if the modal is opened multiple consecutive times, has this init already run, and no focus happens on the following opens?

Reply:
Looks like this works. Nice! :)